### PR TITLE
[wasm] Emit more precise signatures for icalls.

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

File: mcs/tools/wasm-tuner/tuner.cs [hunk 1]
```diff
@@ -123,8 +124,12 @@ public class WasmTuner
 			return "void";
 		else if (t.Name == "Double")
 			return "double";
-		else if (t.Name == "Float")
+		else if (t.Name == "Single")
```
Comment (jaykrell) [inline mcs/tools/wasm-tuner/tuner.cs]: nit: `switch`?

**Comment by vargaz** [comment]
Depends on:
https://github.com/mono/mono/pull/14493

**Comment by vargaz** [comment]
@monojenkins squash

**Comment by vargaz** [comment]
@monojenkins build failed

**Pull request merged**